dynamic-property-jackson/build.gradle.kts

```diff
@@ -5,12 +5,12 @@ plugins {
 
 dependencies {
 
-    compile(project(":dynamic-property-api"))
+    implementation(project(":dynamic-property-api"))
 
-    compile(Libs.jackson_core)
-    compile(Libs.jackson_databind)
-    compile(Libs.jackson_jsr310)
-    compile(Libs.jackson_module_kotlin)
+    implementation(Libs.jackson_core)
+    implementation(Libs.jackson_databind)
+    implementation(Libs.jackson_jsr310)
+    implementation(Libs.jackson_module_kotlin)
```

Comment:
API of this module is `JacksonDynamicPropertyMarshaller` with no-args constructor and two public methods which are independent of these libraries. So I think we should use the `implementation` configuration to avoid redundant transitive dependencies